merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -31,36 +30,15 @@ export class Constraint {
    * @param activityType2
    * @constructor
    */
-  public static ForbiddenActivityOverlap(activityType1: Gen.ActivityType, activityType2: Gen.ActivityType): Constraint {
-    return Constraint.ForEachActivity(
+  public static ForbiddenActivityOverlap(activityType1: Gen.ActivityType, activityType2: Gen.ActivityType): Spans {
+    return Spans.ForEachActivity(
         activityType1,
-        activity1 => Constraint.ForEachActivity(
+        activity1 => Spans.ForEachActivity(
             activityType2,
-            activity2 => Windows.All(activity1.window(), activity2.window()).invert()
+            activity2 => Windows.All(activity1.window().windows(), activity2.window().windows()).invert().spans()
```

Comment:
We're gonna need some more operations for Spans so we can do this without converting to Windows and back. As-is this will lose the activity metadata.

Not your concern for this PR though, just a note for the future